app/utils/GradleRepositoryParser.scala

```diff
@@ -53,7 +53,8 @@ object GradleRepositoryParser extends AbstractParser {
 
     val dependencies = buildFiles
       .map(getDependencies(_, folder, springBootDefaultData, springBootMasterData, defaultProperties))
-      .reduce((r1, r2) => (r1._1 ++ r2._1, r1._2 ++ r2._2))
+      .reduceOption((r1, r2) => (r1._1 ++ r2._1, r1._2 ++ r2._2))
```

Comment:
I'm not sure to understand why you need a reduce option here. The reduce is done on buildFiles and mapped to a pair. It cannot be empty (see canProcess)

Reply:
oki thanks phi, modified